build/yamls/base/crds.yml

```diff
@@ -240,6 +244,9 @@ spec:
             - priority
           type: object
           properties:
+            tier:
+              type: string
+              pattern: '\bEmergency|\bSecurityOps|\bNetworkOps|\bPlatform|\bApplication'
```

Comment:
sorry I'm late to the party, but I have seen the following syntax used in other CRDs:
```
tier:
  type: string
  enum: ["Emergency", ...]
```
Could we use `enum` for our CRDs if possible? I find it easier to read, and error messages that occur during validation may be more helpful if we use an enum instead of a regex?

Reply:
will move to enum